lib/plugins/aws/package/compile/layers/index.js

```diff
@@ -56,11 +56,16 @@ class AwsCompileLayers {
 
     if (layerObject.allowedAccounts) {
       layerObject.allowedAccounts.map(account => {
+        let parsedAccount = account;
+        // cast to string if account is number
+        if (typeof account === 'number' && !isNaN(account)) {
```

Comment:
Could we simplify this via https://lodash.com/docs/4.17.10#isNumber?

Reply:
Sure, I can change it but there is an edge case (that's why there are two conditions here):
```
> _.isNumber(NaN)
true
```
Probably this case is never going to happen but I'd rather be safe than sorry 😄 

I can change it though if you are OK with that kind of regression

Reply:
Ah. Nice catch! 👍

In that case I'd vote for keeping it. The comment above should make clear what's going on here. Thanks for clarifying @sdomagala 👍